lib/buildTool/protractor/testBrowser/templates/protractor.conf.js.tpl

```diff
@@ -58,9 +58,10 @@ var config = {
 
 
   onPrepare: function() {
+    <% if (buildJS.framework[0] && buildJS.framework[0].toLowerCase().search('angular') === -1) { %>
```

Comment:
Hum I though the `.toLowerCase()` would be preventing that. When I tried locally it was working fine :/